clients/venice-client/src/main/java/com/linkedin/venice/fastclient/meta/AbstractStoreMetadata.java

```diff
@@ -23,8 +24,10 @@ public abstract class AbstractStoreMetadata implements StoreMetadata {
 
   public AbstractStoreMetadata(ClientConfig clientConfig) {
     this.instanceHealthMonitor = new InstanceHealthMonitor(clientConfig);
-    if (clientConfig.getClientRoutingStrategy() != null) {
-      this.routingStrategy = clientConfig.getClientRoutingStrategy();
+    ClientRoutingStrategyType clientRoutingStrategyType = clientConfig.getClientRoutingStrategyType();
```

Comment:
Let this return `LEAST_LOADED` as default strategy if its not explicitly configured and probably a switch case to setup `this.routingStrategy`. What do you think?

Reply:
Sure, let me make that change.